src/app/shared/services/auth.service.ts

```diff
@@ -46,11 +51,25 @@ export class AuthService {
 
   login(user: User): Observable<boolean> {
     const users = this.users.getValue();
-    const item = !!users.find((i) => (i.email === user.name || i.name === user.name) && i.password === user.password);
-    if (!item) {
+    const authUser = users.find((i) => (i.email === user.name || i.name === user.name) && i.password === user.password);
+    if (authUser === undefined) {
       return of(false);
     }
+
+    this.authorizedUser.next(authUser);
     this.authorized.next(true);
     return of(true);
   }
+
+  delete(): void {
+    const users = this.users.getValue();
+    const authU = this.authorizedUser.getValue();
```

Comment:
**do not use abbreviated variable names**

Reply:
Done